pycytominer/cyto_utils/cells.py

```diff
@@ -494,7 +542,9 @@ def aggregate_profiles(
         for compartment in self.compartments:
             if compartment_idx == 0:
                 aggregated = self.aggregate_compartment(
-                    compartment=compartment, compute_subsample=compute_subsample
+                    compartment=compartment,
+                    compute_subsample=compute_subsample,
+                    compute_counts=True,
```

Comment:
Is this always going to be True? I am usually in favor of keeping the default behavior the most simple as possible. Is it true that adding True would be more complex? Maybe we always want to compute counts though?

Reply:
I guess my goal is to always compute counts and make the two new metadata features part of all profiles generated. In JUMP we wanted to know the values of these two counts often and going back to the sqlite made it difficult. So I decided to add these values to the profiles itself. It is possible that for site-wise or single cell profiles, these metrics may not be meaningful. That would argue against not having `compute_counts` as True always. What do you think?

Reply:
Cool, thanks for describing. In my view, this is the biggest change introduced in this PR! After merging, pyctominer will forever generate a `Metadata_fields_count` column. I think that this behavior is fine, good behavior even if JUMP requires it often, so I am inclined to agree with keeping `compute_counts` as True.

One other question comes to mind with this default behavior. Should we be adding other metadata fields by default? Cell count is one that immediately comes to mind. I think any discussion of this is beyond scope of the present PR, so I opened #130

Reply:
just to confirm, `compute_counts` defaults to True only in the first pass through aggregate_compartments. This is intended since you only need it once per image.csv, correct?

Reply:
> just to confirm, compute_counts defaults to True only in the first pass through aggregate_compartments. This is intended since you only need it once per image.csv, correct?

That's correct

Reply:
> One other question comes to mind with this default behavior. Should we be adding other metadata fields by default? Cell count is one that immediately comes to mind. I think any discussion of this is beyond scope of the present PR, so I opened #130

I also have added a new field called `Metadata_Object_Count` which counts the number of cells. But I am split on how to proceed. 

I wanted to change it to `Metadata_Cell_Count` as that is essentially what is being calculated. But since `compute_counts` is set to True only for the first pass through `aggregate_compartments()`, if the order of the compartments change for some reason, then `Metadata_Cell_Count` won't be appropriate name for the field. `Metadata_Object_Count` is general enough that it would work.

I don't like the idea of having a single field for counting the number of objects given Shantanu's comment (https://github.com/cytomining/pycytominer/pull/129#issuecomment-7993708690). If the counts in each compartment can sometimes vary, then we may want to have multiple object count fields (one per compartment). I don't particularly like this option either because we are introducing too many new fields. But if this is how we want to proceed then I can make peace with it.

Any thoughts @gwaygenomics?

Reply:
I think it is ok to have multiple metadata columns - remember, "measure everything" :) I'd add a subtitle to the lab's motto though "Measure everything... using sound data science practices ensuring data reproducibility and provenance" (but this doesn't have as nice a ring to it though 😦 )

If you decide to go this route @niranjchandrasekaran (I do think it is up to you to decide), maybe the correct column name is `f"Metadata_Object_Count_{compartment}"`.

If you do not decide to go this route, I think it is at least worth summarizing this discussion in #130